MetaMask/metamask-extension
# feat: move MMI to new asset page

## **Description**

Enables the new asset page for MMI.

MMI has a different set of call to action buttons, but the design is same otherwise.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23588?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Load MMI build
2. Click on native/erc20 assets from the home page
3. Verify price chart + balance are correct
4. Verify all buttons work

## **Screenshots/Recordings**



### **Before**



### **After**

<img width="353" alt="Screenshot 2024-03-19 at 11 16 44 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/64e3e1d9-99d3-4241-b87b-93d6990a5cd6">

<img width="354" alt="Screenshot 2024-03-19 at 11 18 30 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/31e08269-e3f2-47b8-8c1c-3beec79a8ad5">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
